docs/dev/table/catalog.zh.md

```diff
@@ -186,53 +187,57 @@ Currently `HiveCatalog` supports most Flink data types with the following mappin
 | MULTISET      |  N/A |
 
 
-Note that we only cover most commonly used data types for now.
+请注意，我们现在只涵盖最常用的数据类型。
+
+Hive的数据类型中的以下限制会影响Flink和Hive之间的映射：
+
+\* 最小长度 255
 
-The following limitations in Hive's data types impact the mapping between Flink and Hive:
+\* 最大长度 65535
 
-\* maximum length is 255
+## Hive兼容性
 
-\** maximum length is 65535
+有关Hive兼容性和版本，请参阅 [Hive Compatibility]({{ site.baseurl }}/dev/batch/hive_compatibility.html)
 
 
-Catalog Registration
+目录注册
 --------------------
 
 ## Register Catalog in Table API
 
-To register a catalog in Table API, users can create a catalog instance and register it through `TableEnvironment.registerCatalog(name, catalog)`.
+要在Table API中注册目录，用户可以创建目录实例并通过`TableEnvironment.registerCatalog（name，catalog）`注册它。
 
 ## Register Catalog in SQL CLI
 
-To use pre-defined catalogs (`GenericInMemoryCatalog` and `HiveCatalog`) in SQL CLI, please refer to [SQL Clinet]({{ site.baseurl }}/dev/table/sqlClient.html)
+要在SQL CLI中使用预定义的目录（`GenericInMemoryCatalog`和`HiveCatalog`），请参阅 [SQL Clinet]({{ site.baseurl }}/dev/table/sqlClient.html)
 
-To use custom catalogs in SQL CLI, users should develop both a catalog and its corresponding catalog factory by implementing `Catalog` and `CatalogFactory` interfaces respectively.
+要在SQL CLI中使用自定义目录，用户应分别通过实现`Catalog`和`CatalogFactory`接口来开发目录及其相应的目录工厂。
 
-The catalog factory defines a set of properties for configuring the catalog when SQL CLI bootstraps. The set of properties will be passed to a discovery service where the service tries to match the properties to a `CatalogFactory` and initiate an corresponding catalog instance.
+目录工厂定义了一组属性用于在SQL CLI引导时配置目录。 该组属性将传递给发现服务，这些服务会尝试将属性与`CatalogFactory`匹配并启动相应的目录实例。
 
 
 {% top %}
 
-Catalog Modules
+目录模块
 ---------------
 
-`GenericInMemoryCatalog` is built into Flink's Table API.
+`GenericInMemoryCatalog`内置于Flink的Table API中。
 
-To use `HiveCatalog` in Flink Table API and SQL, users need to include `flink-connector-hive` jar in their projects.
+要在Flink Table API和SQL中使用`HiveCatalog`，用户需要在他们的项目中包含`flink-connector-hive` jar。
 
 {% highlight xml %}
 <dependency>
-	<groupId>org.apache.flink</groupId>
-	<artifactId>flink-connector-hive_{{ site.scala_version_suffix }}</artifactId>
-	<version>{{site.version}}</version>
+    <groupId>org.apache.flink</groupId>
```

Comment:
revert changes to this code snippet as it doesn't do anything except indentation